content/docs/04-clusters/04-sandbox-clusters.md

```diff
@@ -16,7 +16,7 @@ import Tooltip from "shared/components/ui/Tooltip";
 
 # Sandbox Clusters Overview
 
-Sandbox Clusters are Kubernetes clusters that run within an existing cluster (also known as a Host Cluster) and share selected resources. By default, Sandbox Clusters will use [k3s](https://github.com/k3s-io/k3s) as virtual Kubernetes cluster, which is a highly available, certified Kubernetes distribution designed for production workloads.
+Sandbox Clusters are Kubernetes clusters that run as nested clusters within an existing cluster (also known as a Host Cluster) and share selected resources. By default, Sandbox Clusters will use [k3s](https://github.com/k3s-io/k3s) as virtual Kubernetes cluster, which is a highly available, certified Kubernetes distribution designed for production workloads.
```

Comment:
@jgautam and @tiwarisumit3 are you two able to share more details on what selected resources are? It's a bit vague so if we can help the reader that'd be ideal.

Reply:
This is more details on the virtual clusters: 
https://www.vcluster.com/

Sandbox Clusters are Kubernetes clusters that run as nested clusters within an existing cluster (also known as a Host Cluster) and share host cluster resources. By default, Sandbox Clusters will use [k3s](https://github.com/k3s-io/k3s) as virtual Kubernetes cluster, certified Kubernetes distribution designed for production workloads.

Reply:
@karl-cardenas-coding - Is it like the definition you want it to be more clear?

Reply:
That's better @tiwarisumit3 . Can we say, "share host cluster resources such as CPU, memory, and storage"?

Reply:
sure @karl-cardenas-coding , should be fine.